lighthouse-core/lib/emulation.js

```diff
@@ -8,19 +8,18 @@
 /** @typedef {import('../gather/driver.js')} Driver */
 
 /**
- * Nexus 5X metrics adapted from emulated_devices/module.json
  * @type {LH.Crdp.Emulation.SetDeviceMetricsOverrideRequest}
  */
-const NEXUS5X_EMULATION_METRICS = {
+const MOTOG4_EMULATION_METRICS = {
   mobile: true,
-  screenWidth: 412,
-  screenHeight: 660,
-  width: 412,
-  height: 660,
+  screenWidth: 360,
+  screenHeight: 640,
+  width: 360,
+  height: 640,
   positionX: 0,
   positionY: 0,
   scale: 1,
-  deviceScaleFactor: 2.625,
+  deviceScaleFactor: 2.625, // ??
```

Comment:
what the heck is this @paulirish

Reply:
that should be the device DPI

Reply:
looks like it's 3 for moto g4 

https://github.com/WPO-Foundation/webpagetest/blob/901d2906711c32577f9c8e19603cf491096fcbf8/www/settings/mobile_devices.ini#L9

Reply:
math checks out too `640 * 3 = 1920`, `360 * 3 = 1080`